frontend/src/components/sldCollection/stepFiveVerifyData/DetailComponent.vue

```diff
@@ -0,0 +1,238 @@
+          >
+            mdi-tray-arrow-down
+          </v-icon>
+        </v-col>
+        <v-col
+          cols="4"
+          align-self="end"
+        >
+          <PrimaryButton
+            id="add"
+            secondary
+            text="Add Student"
+            class="mr-1 mb-1"
+            large-icon
+            icon="mdi-plus"
+          />
+
+          <PrimaryButton
+            id="remove"
+            secondary
+            text="Remove"
+            class="mr-1 mb-1"
+            large-icon
+            icon="mdi-trash-can-outline"
+          />
+
+          <PrimaryButton
+            id="edit"
+            secondary
+            text="Bulk Edit"
+            class="mr-1 mb-1"
+            large-icon
+            icon="mdi-pencil-outline"
+          />
+        </v-col>
+      </v-row>
+
+      <v-row>
+        <v-col cols="12">
+          <CustomTable
+            :headers="config.tableHeaders"
+            :data="studentList"
+            :total-elements="totalElements"
+            :is-loading="isLoading"
+            @reload="reload"
+          />
+        </v-col>
+      </v-row>
+    </v-col>
+  </v-row>
+</template>
+
+<script>
+import alertMixin from '../../../mixins/alertMixin';
+import PrimaryButton from '../../util/PrimaryButton.vue';
+import CustomTable from '../../common/CustomTable.vue';
+import ApiService from '../../../common/apiService';
+import {ApiRoutes} from '../../../utils/constants';
+import {isEmpty, omitBy } from 'lodash';
+import {ALERT_NOTIFICATION_TYPES} from '../../../utils/constants/AlertNotificationTypes';
+ 
+export default {
+  name: 'DetailComponent',
+  components: {
+    PrimaryButton,
+    CustomTable
+  },
+  mixins: [alertMixin],
+  props: {
+    config: {
+      type: Object,
+      required: true,
+      default: null
+    }
+  },
+  emits: [],
+  data() {
+    return {
+      chip: true,
+      pageNumber: 1,
+      pageSize: 50,
+      studentList: [],
+      isLoading: false,
+      totalElements: 0,
+      searchText: '',
+      headerSearchParams: {
+        type: '' //this.config.defaultFilter
+      },
+    };
+  },
+  mounted() {
+    this.loadStudents();
+  },
+  created() {
+  },
+  methods: {
+    loadStudents() {
+      this.isLoading= true;
+      ApiService.apiAxios.get(`${ApiRoutes.sld.SLD_SCHOOL_COLLECTION_STUDENT}/${this.$route.params.schoolCollectionID}/paginated`, {
+        params: {
+          pageNumber: this.pageNumber - 1,
+          pageSize: this.pageSize,
+          searchParams: omitBy(this.headerSearchParams, isEmpty),
+        }
+      }).then(response => {
+        this.studentList = response.data.content;
+        this.totalElements = response.data.totalElements;
+      }).catch(error => {
+        console.error(error);
+        this.setFailureAlert({text: error?.response?.data?.message ? error?.response?.data?.message : 'An error occurred while trying to retrieve students list. Please try again later.', alertType: ALERT_NOTIFICATION_TYPES.ERROR});
```

Comment:
for setFailureAlert it just takes message a String. 

You do not need to pass the text/alertType object. 

so this could be this.setFailureAlert('I failed');